src/Cli/dotnet/commands/dotnet-test/TestModulesFilterHandler.cs

```diff
@@ -50,7 +50,7 @@ public bool RunWithTestModulesFilter(ParseResult parseResult)
 
             foreach (string testModule in testModulePaths)
             {
-                var testApp = new TestApplication(testModule, _args);
+                var testApp = new TestApplication(new Module(testModule, null, null), _args);
```

Comment:
I don't like to pass a null, can you expose a new ctor with only the testmodule?